src/bandersnatch/tests/plugins/test_blacklist_name.py

```diff
@@ -191,3 +191,5 @@ def test__plugin__loads__default(self):
         plugins = bandersnatch.filter.filter_release_plugins()
         names = [plugin.name for plugin in plugins]
         self.assertIn("blacklist_release", names)
+
+        # TODO: add test for existing release plugin
```

Comment:
I don't follow this - I could just be slow - What's the existing release plugin?